internals/detect_intent.py

```diff
@@ -86,6 +86,10 @@ def detect_field(subject):
     r'entry on the feature dashboard.*\s+'
     r'[> ]*https?://(www\.)?chromestatus\.com/'
     r'(feature|guide/edit)/(?P<id>\d+)', re.I)
+CHROMESTATUS_LINK_GENERATED_GATE_RE = re.compile(
```

Comment:
You could reduce duplication here by defining a shared pattern string like
```
CHROMESTATUS_LINK_GENERATED_PATTERN = (
    r'Chrome( Platform)? ?Status(.com)?[ \w]*:?\s+'
    r'[> ]*https?://(www\.)?chromestatus\.com/'
    r'(feature|guide/edit)/(?P<id>\d+)')
```
And then use it for CHROMESTATUS_LINK_GENERATED_RE and CHROMESTATUS_LINK_GENERATED_GATE_RE with and appended part for the gate parameter.  You would need to rename `(?P<id>\d+)` for the gate ID to avoid name collision, like `(?P<gate_id>\d+)`.

Reply:
Great suggestion - updated